### ISSUE TITLE
broken roff output

### ISSUE THREAD

**Issue by pawelz** [rtomayko/ronn#34]
Just an example of input file that is not rendered correctly:

```
a(1)
====

* `toggle_status`
  - Toggle the display of the status bar.
* `spawn <executable> <additional args>` TODO explain path-alike expansion
```

I'm getting `warn: unrecognized block tag: "code"` message and broken man output. HTML looks ok.

Note that the following input file renders correctly:

```
a(1)
====

* `abc`
  - def

* `toggle_status`
  - Toggle the display of the status bar.
* `spawn <executable> <additional args>` TODO explain path-alike expansion
```

**Comment by rtomayko** [comment]
Awesome. These examples are hugely useful. Thanks for reporting them.

**Comment by rtomayko** [comment]
fix lists with inline text and then sub-lists, closed by 72e72c14c4bead762f9db7e0d33d32e0d63d8496

**Comment by rtomayko** [comment]
The text comes through in roff output now but there's other issues with nested list indent. I'll try to get that working for the next release.
